mgnify_pipelines_toolkit/analysis/amplicon/assess_inflection_point_mcp.py

```diff
@@ -65,14 +65,18 @@ def assess_inflection_point_mcp_for_sample(_PATH, inf_point_list, rev=False):
 
     read_count = get_read_count(_PATH) # get readcount from fastq
 
+    max_line_count = None
+    if read_count > 300_000:
```

Comment:
Can you create a constants file and put this value there? It's easier to track / update that way